term.go

```diff
@@ -86,8 +87,11 @@ type Terminal struct {
 	printer                Printer
 	cmd                    *exec.Cmd
 	readWriterConfigurator ReadWriterConfigurator
+	onResize               ResizeHandler
 }
 
+type ResizeHandler = func(size *pty.Winsize)
```

Comment:
We should not expose a third party API as a type in our APIs - it assumes that Terminal is always connected to a "github.com/creack/pty"